From 69140573dec8fc99bc92e764ee945748aba27877 Mon Sep 17 00:00:00 2001
From: Alejandro Abdelnur <tucu@apache.org>
Date: Fri, 18 Jan 2013 00:43:28 +0000
Subject: [PATCH 1259/1357] MR1: MAPREDUCE-4923. Add toString method to TaggedInputSplit. (sandyr via tucu)

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/branches/branch-1@1434995 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit bed22b230563228cdc5c6366b0af2e5ab48ae63a)

Reason: For ZD-10823
Author: Sandy Ryza
Ref: CDH-9861
---
 .../apache/hadoop/mapred/lib/TaggedInputSplit.java |    5 +++++
 .../mapreduce/lib/input/TaggedInputSplit.java      |    5 +++++
 2 files changed, 10 insertions(+), 0 deletions(-)

diff --git a/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapred/lib/TaggedInputSplit.java b/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapred/lib/TaggedInputSplit.java
index 7be9ff9..c08bdd9 100644
--- a/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapred/lib/TaggedInputSplit.java
+++ b/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapred/lib/TaggedInputSplit.java
@@ -137,4 +137,9 @@ class TaggedInputSplit implements Configurable, InputSplit {
     this.conf = conf;
   }
 
+  @Override
+  public String toString() {
+    return inputSplit.toString();
+  }
+
 }
diff --git a/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapreduce/lib/input/TaggedInputSplit.java b/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapreduce/lib/input/TaggedInputSplit.java
index 68bb789..08336f8 100644
--- a/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapreduce/lib/input/TaggedInputSplit.java
+++ b/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapreduce/lib/input/TaggedInputSplit.java
@@ -156,4 +156,9 @@ class TaggedInputSplit extends InputSplit implements Configurable, Writable {
     this.conf = conf;
   }
 
+  @Override
+  public String toString() {
+    return inputSplit.toString();
+  }
+
 }
-- 
1.7.0.4

